appinventor/appengine/src/com/google/appinventor/client/TranslationComponentProperty.java

```diff
@@ -115,6 +115,7 @@ public static String getName(String key) {
     map.put("CharacterEncoding", MESSAGES.CharacterEncodingProperties());
     map.put("Checked", MESSAGES.CheckedProperties());
     map.put("ColorChangedEventEnabled", MESSAGES.ColorChangedEventEnabledProperties());
+    map.put("RequestFocus", MESSAGES.RequestFocusProperties());
```

Comment:
I believe this needs to be moved to TranslationComponentMethods.java now that it is a function instead of a Property